### DISCUSSION TITLE
Hotness default setting #8370

### DISCUSSION THREAD

**Discussion by SoskaCream** [bluesky-social/atproto#3129] [Bluesky Client App]
The new setting of the reply sorting by hotness (https://github.com/bluesky-social/atproto/actions/runs/11983027063) has taken a lot of users by surprise, and is not something popular at all, decried as a symptom of enshittification.
Can we get more information on why such decried and infamous feature has been pushed through, and what can be done to rectify this?

**Comment by DavidBuchanan314** [comment]
A change that you don't personally like is not the same thing as "enshittification".

That aside, reply sorting behaviour can be adjusted in your client settings.

**Reply by Jmuccigr** [reply]
I don’t think the OP was expressing a personal opinion, but rather what others are saying.

That said, “the algorithm “ gets a lot of the blame for the problems with that other site, so the addition of one to Bluesky is not an unreasonable concern.
